explain-overview.md

```diff
@@ -6,500 +6,52 @@ aliases: ['/docs/dev/query-execution-plan/','/docs/dev/reference/performance/und
 
 # `EXPLAIN` Overview
 
-Based on the latest statistics of your tables, the TiDB optimizer chooses the most efficient query execution plan, which consists of a series of operators. This document details the execution plan in TiDB.
+Because SQL is a declarative language, it describes what the results of a query should look like, **not the methodology** to actually retrieve those results. TiDB considers all the possible ways in which a query could be executed, including using what order to join tables and whether any potential indexes can be used. The process of _considering query execution plans_ is known as SQL optimization.
```

Comment:
Can we change another description? 
![image](https://user-images.githubusercontent.com/41534398/102179166-aa42cb80-3ee1-11eb-9904-f4e201ad66c3.png)
![image](https://user-images.githubusercontent.com/41534398/102179230-c21a4f80-3ee1-11eb-8e31-85503051269f.png)
Looks like it's quite duplicated.

Reply:
It was intentional to duplicate but cover different perspectives. The SQL Tuning is high level and does not mention EXPLAIN yet. This is a key concept: a lot of naive users believe you can look at a query and tell if its efficient.

Reply:
```suggestion
SQL is a declarative language. It describes what the results of a query should look like, **not the methodology** to actually retrieve those results. TiDB considers all the possible ways in which a query could be executed, including using what order to join tables and whether any potential indexes can be used. The process of _considering query execution plans_ is known as SQL optimization.
```